tpm2/structures.go

```diff
@@ -104,13 +104,18 @@ func decodePublic(in *bytes.Buffer) (Public, error) {
 // RSAParams represents parameters of an RSA key pair.
 //
 // Symmetric and Sign may be nil, depending on key Attributes in Public.
-// Modulus must always be non-nil.
+//
+// One of Modulus and ModulusRaw must always be non-nil. Modulus takes
+// precedence.
```

Comment:
nit: I don't the extra newline is needed here.

Reply:
Done